src/map/script.c

```diff
@@ -22970,6 +22995,70 @@ BUILDIN(navigateto)
 }
 
 /**
+ * Clan System: Add a player to a clan
+ */
+BUILDIN(clan_join)
+{
+	struct map_session_data *sd = NULL;
+	int clan_id = script_getnum(st, 2);
+
+	if (script_hasdata(st, 3))
+		sd = script->charid2sd(st, script_getnum(st, 3));
+	else
+		sd = map->id2sd(st->rid);
+
+	if (sd == NULL)
+		return false;
```

Comment:
This should do a script_pushint(st, false) in this case, so that the script has a feedback (invalid character implies that the join request failed, I suppose)